packages/authhero/src/types/SMSService.ts

```
@@ -0,0 +1,10 @@
+export type SendSMSParams = {
+  to: string;
+  text: string;
+  template: string;
+  options: any;
+  data: Record<string, string>;
+};
+export type SendSMSResponse = {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)